graylog2-web-interface/src/components/indices/data-tiering/types.ts

```diff
@@ -43,6 +43,11 @@ export type DataTieringFormValues = {
   warm_tier_repository_name?: string | null,
 }
 
+export type DataTieringStatus = {
+  has_failed_snapshot: boolean,
+  failed_snapshot_name: string | null,
+}
+
```

Comment:
Ideally, this would also not live in the server, but for the sake of moving this forward let's leave it like this.